tekton/src/pipelines/gitops/deprovision-mas-instance.yml.j2

```diff
@@ -186,3 +192,91 @@ spec:
       taskRef:
         kind: Task
         name: gitops-process-mongo-user
+{% if deprovision_manage_addons == true %}
+    - name: gitops-deprovision-efs
```

Comment:
gitops-deprovision-efs hould be in the deprovison-cluster pipeline I would have thought? We don't want to remove EFS from the cluster that any existing MAS instances might stil be using.

Reply:
gitops-provision-efs creates a configurable storage for each mas instance. 
similarly gitops-deprovision-efs deletes the configurable storage at mas instance level.  
Hence include deprovision efs is included in deprovision-mas-instance.yml.j2

Reply:
It is an EFS per MAS instance including the storage class, my mistake.